### DISCUSSION TITLE
Compress to CHD support

### DISCUSSION THREAD

**Discussion by msauder** [aonez/Keka#1462] [Ideas]
With emulators on iOS, this would be hugely beneficial. After reading MAME documentation I'm not sure how easy it would be to implement.

**Comment by gingerbeardman** [comment]
Oh this is a fun idea. 

I often do CHD using [chdman](https://github.com/charlesthobe/chdman) (binary is part of MAME download).
It would require Keka to accept valid BIN/CUE pairs.

We could also add compressed iso for PSP using [ciso](https://github.com/jamie/ciso) but that is much more niche.
